src/interface/report/Results/Character/PlayerInfo/Gear.tsx

```diff
@@ -11,16 +11,15 @@ interface Props {
 
 const Gear = (props: Props) => {
   const { gear } = props;
-  const EPIC_ITEMS_ILVL = 340;
+  const EPIC_ITEMS_ILVL = 200;
```

Comment:
I won't stop the PR but this should probably be placed somewhere else since it is used in multiple locations

Reply:
Also this one is more of a blocker isn't epic items purple items? Meaning 184 would be the item level threshold not 200? (mythic dungeons)

Reply:
Darkmoon decks are 200 and blue

Reply:
Donesies

Reply:
I know you did done it but It feels like Dark moon decks are the exception not the standard :P (which i guess is what you thought)